features/organized-chaos.js

```diff
@@ -0,0 +1,6 @@
+setTimeout(() => {
+  javascript:(function(){['', '-ms-', '-webkit-', '-o-', '-moz-'].map(function(prefix){document.body.style[prefix + 'transform'] = 'rotate(180deg)';});}());
```

Comment:
```suggestion
  document.body.style.transform = 'rotate(180deg)';
```
Or even better, but use a CSS file. Tested on Chromium. I couldn't get the extension to load on Firefox.

Reply:
Ste isn't fully supported on Firefox yet